evm_arithmetization/src/generation/prover_input.rs

```diff
@@ -59,8 +59,15 @@ impl<F: Field> GenerationState<F> {
         }
     }
 
-    fn no_txn(&mut self) -> Result<U256, ProgramError> {
-        Ok(U256::from(!self.inputs.has_txn as u8))
+    fn run_end_of_txns(&mut self) -> Result<U256, ProgramError> {
+        self.jumpdest_table = None;
```

Comment:
`run_end_of_txns` determines whether we have reached the end of the transaction length. SInce we need to reset the jumptable at some point before the previous transaction, I made use of this `PROVER_INPUT` for that.